src/list/manage/components/edit-item-details.js

```diff
@@ -59,9 +62,10 @@ export default class EditItemDetails extends React.Component {
   }
 
   render() {
-    const {fields: { title }, onSave, onCancel, onDelete, onReveal} = this.props;
+    const {fields: { title }, onSave, onCancel, onDelete, onReveal, error} = this.props;
     const {itemId, ...saveState} = this.state;
     const newItem = itemId === null;
+    const isDuplicate = error && !!~error.message.indexOf("This login already exists");
```

Comment:
it would be nice if we could better differentiate this error before it's thrown to the extension.

nit: prefer `...indexOf(...) === 0` over `!!~...`; the former, while more verbose, is also more explicit

Reply:
moved comment on differentiation to be a requirement for #129